includes/reports/overview/data-provider/class-sensei-reports-overview-data-provider-lessons.php

```diff
@@ -64,6 +64,12 @@ public function get_items( array $filters ): array {
 		if ( isset( $filters['search'] ) ) {
 			$lessons_args['s'] = $filters['search'];
 		}
+
+		if ( ! empty( $filters['orderby'] ) && ! empty( $filters['order'] ) ) {
+			$lessons_args['orderby'] = $filters['orderby'];
+			$lessons_args['order']   = $filters['order'];
+		}
```

Comment:
The same applies here.

Reply:
[Removed it](https://github.com/Automattic/sensei/pull/4970/commits/a411bd74121681db267ee86ce3838b70265feec6)